src/toil_scripts/lib/__init__.py

```diff
@@ -1,4 +1,3 @@
-# FIXME: replace with bd2k.util.iterables.flatten
 def flatten(x):
```

Comment:
Hm, I can't get your version of `flatten` to perform what the current version does:

Common lib version:
```
In [43]: flatten( ('cat',('foo', 'bar'),[[['tacobell']]] ))
Out[43]: ['cat', 'foo', 'bar', 'tacobell']
``` 

BPL version:
```
from bd2k.util.iterables import concat

def flatten(x):
    return list(concat(x))
In [48]: flatten( ('cat',('foo', 'bar'),[[['tacobell']]] ))
Out[48]: ['cat', ('foo', 'bar'), [[['tacobell']]]]
```
Other version you posted:
```
from bd2k.util.iterables import flatten as bpl_flatten

def flatten(x):
    return list(bpl_flatten((x,))
In [51]: flatten( ('cat',('foo', 'bar'),[[['tacobell']]] ))
Out[51]: ['cat', ('foo', 'bar'), [[['tacobell']]]]
```